shared/css/conversation.css

```diff
@@ -538,8 +592,14 @@ body[platform="win"] .share-service-dropdown.overflow > .dropdown-menu-item {
   height: 150px;
 }
 
+.media-wrapper > .local .avatar-volume,
+.media-wrapper > .remote .avatar-volume{
+  width: 106px;
+  height: 106px;
+}
+
 .media-wrapper > .local .local-video,
-.media-wrapper > .focus-stream > .local .local-video {
+.media-wrapper > .remote.focus-stream > .local .local-video {
```

Comment:
This feels like a wrong change. The focus stream is not always the remote stream - sometimes its the screen sharing stream (especially on standalone). This feels like its breaking something.